batchflow/models/torch/vnet.py

```diff
@@ -0,0 +1,96 @@
+"""  Milletari F. et al "`V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
+<https://arxiv.org/abs/1606.04797>`_"
+"""
+from .encoder_decoder import EncoderDecoder
+from .blocks import ResBlock
+
+
+
+class VNet(EncoderDecoder):
+    """ VNet-like model.
+
+    Parameters
+    ----------
+    build_from_stages : bool
```

Comment:
Do we really need this?

Reply:
Same logic is implemented in `UNet.build_config`, I just cleared it a bit, as @SergeyTsimfer asked in https://github.com/analysiscenter/batchflow/pull/462#discussion_r359054991. IMO, it's a useful param, that allows writing short configs in notebooks (i.e. `{'body/encoder/num_stages': n, 'build_from_stages': True}`), when everything we want is to increase network depth.

Reply:
Within `build_config` one can easily check for `num_stages` within `self.config`. Which is essentially the same as having `build_from_stages ` without `build_from_stages`, isn't it?

Reply:
Not exactly. Since default and outer configs are always combined (see [`TorchModel.build`](https://github.com/analysiscenter/batchflow/blob/master/batchflow/models/torch/base.py#L313)), `num_stages` will always be in `self.config` (as well as the other default params). And therefore from inside of `build_config` it's impossible to distinguish between the case when `num_stages` is actually the only param that we mentioned in outer config (and want to build other params from it) and the case when along with `num_stages` we also pass something else (`filters`, `layout`, etc).

Reply:
Not exactly. Since default and outer configs are always combined (see [`TorchModel.build`](https://github.com/analysiscenter/batchflow/blob/master/batchflow/models/torch/base.py#L313)), the same `num_stages` value (if passed through outer config) will always present in both `self.config` and `self.full_config`. And therefore, _if we want_ to have functionality of so-called 'building from stages', we need to distinguish from inside of `build_config` between two cases:
1. `num_stages` is actually the only param that we mentioned in outer config;
2. along with `num_stages` some other params (`filters`, `layout`, etc) were mentioned.

We can achieve that by either checking for missing params in `self.config` (almost like in [earlier version](https://github.com/analysiscenter/batchflow/blob/b4c7b44ee23f4e8da5008c79327c22de1df13b2a/batchflow/models/torch/vnet.py#L77)) or via notorious `build_from_stages`.